lms/djangoapps/courseware/views/views.py

```diff
@@ -520,15 +519,11 @@ def course_about(request, course_id):
             if professional_mode.bulk_sku:
                 ecommerce_bulk_checkout_link = ecomm_service.checkout_page_url(professional_mode.bulk_sku)
 
-        # We need to look up the price from the CourseMode only when the EcommerceService is enabled OR
-        # the legacy shoppingcart ecommerce implementation is enabled.
-        registration_price = 0
-        if _is_ecomm_service_enabled or _is_shopping_cart_enabled:
-            registration_price = CourseMode.min_course_price_for_currency(
-                course_key,
-                settings.PAID_COURSE_REGISTRATION_CURRENCY[0]
-            )
-
+        # Find the minimum price for the course across all course modes
```

Comment:
Checking to see if the ecommerce backend is enabled does save you a DB call. I'm guessing that is why the original check was put in place.